public/index.html

```diff
@@ -4,7 +4,7 @@
         <title>Jared Griffin</title>
     </head>
     <body>
-        <div id="app"/>
-        <script src="./bundle.js" type="text/javascript"></script>
+        <div id="app"><div/>
+        <script src="http://localhost:7700/public/bundle.js" type="text/javascript"></script>
```

Comment:
💡 When we deploy to the cloud, this will have to be parameterized. This will only work locally as is 😞